windows/src/README.md

```diff
@@ -1,85 +1,108 @@
 
-Release build prerequisites
----------------------------
+### Release build prerequisites
 
-1. Install 7-Zip 64-bit (or 32-bit on x86 Windows) http://www.7-zip.org/ (used for archiving build files -- may be eliminated in future)
-2. Install HTML Help Workshop from https://www.microsoft.com/en-us/download/details.aspx?id=21138
-3. Run `make test-certificate` to create a test certificate, or else create `UserDefines.mak` to locate code signing certificates
-4. Install WiX to c:\program files (x86)\windows installer xml v3.5 https://wix.codeplex.com/releases/view/60102
+For local development you do not need to perform a release build so these are optional.
 
-Building Keyman Desktop and Keyman Developer
-============================================
+1. Install [7-Zip](http://www.7-zip.org/) 64-bit (or 32-bit on x86 Windows). 7-Zip is used for archiving build files -- may be eliminated in future.
+2. Install [HTML Help Workshop](https://www.microsoft.com/en-us/download/details.aspx?id=21138)
+4. Install [WiX](https://wix.codeplex.com/releases/view/60102) to **C:\Program Files (x86)\Windows Installer XML v3.5**
 
-1. Start a `VS2017 x64_x86 Cross Tools` command prompt.
-2. Run `make build` from the folder `windows/src` folder.
-3. Artifacts from a successful build will be placed in `windows/bin` folder.
+## Building Keyman Desktop and Keyman Developer
+
+1. Start 'x64_x86 Cross Tools Command Prompt for VS 2017'.
+2. Run `make build` from the **windows/src** folder.
+3. Artifacts from a successful build will be placed in **windows/bin** folder.
 
 Type `make` to see build targets. Common build targets are:
 
   `make build` <-- builds Keyman Desktop and Keyman Developer
   `make clean` <-- remove temporary files and build artifacts
   `make release` <-- makes a release of all Keyman Windows projects
 
-Release build
--------------
+By default the version generated will be 10.0.700.0.
+
+### Release build
 
-1. Start a `VS2017 x64_x86 Cross Tools` command prompt.
-2. Run `make release` from the folder `windows/src` folder.
-3. Artifacts from a successful build will be placed in `windows/release` folder.
+To perform a release build, you will need to obtain valid certificates. A release build is
+unnecessary for local development.
+
+1. Start 'x64_x86 Cross Tools Command Prompt for VS 2017'.
+2. Run `make release` from the **windows/src** folder.
+3. Artifacts from a successful build will be placed in **windows/release** folder.
 
 Note: by default, the version generated will be 10.0.700.0. You will not be able to
 install it over a later version of Keyman, and will need to uninstall and reinstall.
 
-Detailed Preqrequisite Notes
-============================
+## Installing Keyman
+
+These steps are only required the first time you install Keyman:
+1. Install release versions of Keyman 10.0 and Keyman Developer 10.0.
+    * Download the [alpha versions](https://keyman.com/beta/) and run the installers.
```

Comment:
Do you want to install [release (stable)](https://keyman.com/downloads/) or [pre-release](https://keyman.com/beta/) versions of Keyman 10.0 and Keyman Developer 10.0?

Reply:
Let's leave out the link to [alpha versions] and just point them to Desktop/Developer paths. The developer needs to find the appropriate installer. Otherwise, we have to remember to update this come release. Say instead, `Install latest official versions of Keyman Desktop and Keyman Developer from https://keyman.com` or something like that.

Reply:
Sure I can change that. I linked to the alpha version since the latest release is 9.0. Would the latest release installer still work?

Reply:
Yeah, we should probably go with 'Latest official build (alpha)'. That will be reasonably accurate for anyone working off `master` branch.

Older versions will end up going in different places and/or have breaking compat issues with COM API etc.